[05] Rock Paper Scissor/Rock Paper Scissor/Rock Paper Scissor/RPSEntireGameView.swift

```diff
@@ -27,7 +30,15 @@ struct RPSEntireGameView: View {
 					RPSGameOverView(game: $currentGame)
 				}
 				Spacer()
-				RPSScoreView(score: currentGame.userScore)
+				HStack {
+					gameplayInfoButton
```

Comment:
This is a good move! My appreciation to my guy Murat again for his suggestion! 💪